dds_cli/account_manager.py

```diff
@@ -48,10 +48,10 @@ def __init__(
         if self.method not in ["add", "delete", "revoke"]:
             raise dds_cli.exceptions.AuthenticationError(f"Unauthorized method: '{self.method}'")
 
-    def add_user(self, email, role, project, no_mail=False):
+    def add_user(self, email, role, project, unit, no_mail=False):
```

Comment:
I think you should specify a default here, possibly an empty string `unit=""`. The reason is, that also the CLI function `main.py:grant_project_access` also calls the `add_user` method: 

https://github.com/ScilifelabDataCentre/dds_cli/blob/b38634da3f865ccdb8522ad7e6b2e1c9cb3ca07a/dds_cli/__main__.py#L949

Reply:
Fixed!